resources/jaeger/README.md

```diff
@@ -6,12 +6,7 @@
 ## Details
 The Envoy sidecar uses Jaeger to trace the request flow in the Istio service mesh. Istio and Envoy use the Zipkin protocol to communicate with the tracing back-end. Jaeger provides compatibility with the Zipkin protocol. This allows you to use Zipkin protocol and clients in Istio, Envoy, and the Kyma services.
 
-For more details, see the [Istio Distributed Tracing](https://istio.io/docs/tasks/telemetry/distributed-tracing.html) documentation.
 
-## Installing Jaeger locally
-Jaeger installation is optional, and you cannot install it locally by default. However, you can install it on a Kyma instance and run it locally using Helm.
+## Installation
+While Jager installs automatically during cluster installation, local Jaeger installation is optional. In this case, you can install Jaeger on a Kyma instance and run it locally using Helm. For details, see [Install Jaeger locally](../../docs/tracing/docs/010-details-Jaeger.md).
```

Comment:
Maybe you can change `In this case` into sth more specific as it might be unclear which scenario you refer to here?